packages/plugins/rubygem-foreman_remote_execution/rubygem-foreman_remote_execution.spec

```diff
@@ -1,15 +1,18 @@
 # template: foreman_plugin
 %{?scl:%scl_package rubygem-%{gem_name}}
 %{!?scl:%global pkg_name %{name}}
+%{!?_root_sbindir:%global _root_sbindir %{_sbindir}}
+%{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
 
+%global debug_package %{nil}
```

Comment:
For my understanding: why is this needed?

Reply:
`noarch` packages should not produce any debug symbols, so it should not be needed. but with el8 you never know.